_data/en/settings.yml

```diff
@@ -228,7 +228,7 @@ nav:
   anchor_to_top: Back to top
   become_a_partner: Become a partner
   contact_us:
-    title: Business inquiries
+    title: Contact us
```

Comment:
This string is reused in many places across both the main site and partner subsite, hence the unintended change to the help center and main site footer. We'll probably want to create a separate string for "Business inquiries" and use it where appropriate.